### DISCUSSION TITLE
Quality Selector/Bitrate Switcher for Video Component

### DISCUSSION THREAD

**Discussion by johnpeeke** [Megabit/Blazorise#4881] [Q&A] [answered]
I might be blind, but is there a quality selector button available for the video player? I use it for HLS streaming and would like to have the option to change the quality of the video.

**Comment by stsrki** [comment, marked as answer]
It seems this option is missing from the API. We will try to add it for next 1.3 release.

**Reply by johnpeeke** [reply]
Thanks for the update.

**Reply by stsrki** [reply]
Working on the feature for 1.3, #4883
